ZA2-Riyaaz Singh - JavaScript-Core-2-Coursework-Week2

Riyaazct opened the pull request:
<!--

The title for your pull request should be made in this format

CITY CLASS_NO - FIRST_NAME LAST_NAME - MODULE - WEEK_NO

For example,

London Class 7 - Chris Owen - HTML/CSS - Week 1

Please complete the details below this message

-->

**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Riyaaz Singh
- Your City: Cape Town
- Your Slack Name: Riyaazct

# Homework Details

- Module: 2
- Week: 2

# Notes

- What did you find easy?
- everything after I got over the initial struggle.

- What did you find hard?
- looping over the objects initially and applying array methods and basically everything was hard in the first few challenges in the exercises of the mandatory folder.

- What do you still not understand?


- Any other notes?

ezekiel444 reviewed (commented):
F-highlight-words not meeting the requirement(highlight the selected word) because you're assigning a background color to an undefined spanTag line 33.
todolist is accepting empty value and impossible to delete with an empty value.

Riyaazct committed:
update exercises folder

Riyaazct committed:
Update exercise-1.js

Riyaazct committed:
Update exercise-2.js

Riyaazct committed:
Update exercise-3.js

Riyaazct committed:
Update exercise-1.js

Riyaazct committed:
Update exercise-2.js

Riyaazct committed:
Update 1-writers.js

took me almost an hour to figure out that I omitted the letter 'a' before 'occupation'

Riyaazct committed:
Update 1-writers.js

Riyaazct committed:
Update 2-water-bottle.js

Riyaazct committed:
update exercise

Riyaazct committed:
Update 3-groceries.js

Riyaazct committed:
Update 3-groceries.js

Riyaazct committed:
Update 3-groceries.js

Riyaazct committed:
Update 3-groceries.js

Riyaazct committed:
Update 4-people-i-know.js

Riyaazct committed:
Update 4-people-i-know.js

Riyaazct committed:
Update 5-reading-list.js

Riyaazct committed:
Update 6-budgets.js

Riyaazct committed:
Update 7-eligible-students.js

Riyaazct committed:
Update 8-journey-planner.js
